src/documentation/reference/errors/result-codes/operation-specific/create-account.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Create Account Result Codes
+order: 10
+---
+
+These are result codes that communicate success (200) or failure (400) specific to the `Create Account` operation, which often fails because the new account does not meet the minimum reserve.
+
+Learn more about the [`Create Account` operation](../../../glossary/operations/create-account.mdx).
+
+<NestedTable>
+
+- RESULT CODE
+  - STELLAR PROTOCOL CODE
+  - DESCRIPTION
+- OpSuccess
+  - CREATE_ACCOUNT_SUCCESS
+  - The inner object result is valid and the operation was a success.
+- OpMalformed
+  - CREATE_ACCOUNT_MALFORMED
+  - There are too few valid signatures, or the transaction was submitted to the wrong network.
+- OpUnderfunded
+  - CREATE_ACCOUNT_UNDERFUNDED
+  - The source account was not found.
+- OpLowReserve
+  - CREATE_ACCOUNT_LOW_RESERVE
+  - The operation is not supported at this time.
+
+</NestedTable>
+
+<ExampleResponse title="Example Response for an “op_low_reserve” Result Code">
```

Comment:
We don't have this yet. Can we use `<MethodTable/>` for now

Reply:
yes